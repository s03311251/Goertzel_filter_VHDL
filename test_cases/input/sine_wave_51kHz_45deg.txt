36A0
3C99
3FAA
3F81
3C22
35E7
2D71
239C
1969
0FE2
07FF
028D
001B
00E8
04E0
0B9A
1468
1E64
288A
31D1
3948
3E2C
3FFE
3E8D
3A01
32CE
29B2
1F99
158A
0C8C
0588
0137
0008
021A
0737
0EDA
183C
2269
2C56
3502
3B8A
3F45
3FD0
3D1F
3776
2F6A
25CC
1B97
11D5
0984
037D
005D
0076
03C4
09F3
1260
1C30
2663
2FF0
37DE
3D5D
3FDF
3F23
3B3A
348D
2BC7
21CE
17A7
0E59
06D7
01E4
0003
0163
05E0
0D07
161C
2033
2A45
334A
3A59
3EBA
3FF9
3DF7
38E8
3150
27F5
1DCA
13D9
0B24
048F
00C5
0029
02CB
0866
1068
1A01
2435
2DFC
3657
3C6B
