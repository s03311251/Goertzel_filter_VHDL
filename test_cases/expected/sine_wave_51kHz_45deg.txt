FFF70EA0
FFF1FBE0
